lib/message_bus/http_client/configuration.rb

```diff
@@ -0,0 +1,40 @@
+module MessageBus
+  class HTTPClient
+    module Configuration
```

Comment:
What is the benefit of this being a separate module if it is only used once?

Reply:
It makes the main file easier to read and I avoid having to flood the main class with more comments for yard.

Reply:
But all of these options are important to know about when instantiating the class. Just seems like unnecessary indirection.